docs/ARM.md

```diff
@@ -367,3 +367,60 @@ Outputs the details of an ARM server.
 ```
 knife azurerm server show MyVMName --azure-resource-group-name MyResourceGrpName -c ~/.chef/knife.rb
 ```
+
+#### `cloud-api` bootstrap feature
```

Comment:
In ARM, user doesn't have to pass `cloud-api` separately. Bootstrap happens by default using the extension only. So this description may require some changes here.
Sorry I didn't point this earlier.